config/samples/nifi_v1alpha1_nificonnection.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: nifi.konpyutaika.com/v1alpha1
+kind: NifiConnection
+metadata:
+  name: nificonnection-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
Could you add a complete and documented example?

Reply:
done